src/vs/workbench/browser/workbench.contribution.ts

```diff
@@ -49,6 +49,11 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
 				'description': localize('showEditorTabs', "Controls whether opened editors should show as individual tabs, one single large tab or if the title area should not be shown."),
 				'default': 'multiple'
 			},
+			'workbench.editor.showEditorActionsInTitleBar': {
```

Comment:
I was also think of this. But we would have to make it an enum setting as users might switch between multiple/single and no tabs and would expect it to not always be in the title bar. Also, I want to make this on by default when tabs are hidden.
How about we make that change in a different PR? I can change this to an enum setting for the current behaviour and we can expand the settings in a different PR. What do you think?

Reply:
Changed it, not sure about the exact naming of the enums yet

Reply:
Hm, not sure I understand the need for an enum: I was suggesting that its a `boolean` and when enabled, editor actions show in the main title and hide in the editor title. This works irrespective of how tabs are configured to be visible.

Reply:
I want this to be an opt-out feature when tabs are hidden. This would not be possible with a boolean value (as the default value is true and suddenly all users would have the actions in the title bar).